onnxruntime/core/providers/dml/DmlExecutionProvider/src/Operators/DmlOperatorResize.cpp

```diff
@@ -289,7 +294,12 @@ class DmlOperatorResize : public DmlOperator, public ResizeHelper
         std::vector<DML_TENSOR_DESC> inputDescs = GetDmlInputDescs();
         std::vector<DML_TENSOR_DESC> outputDescs = GetDmlOutputDescs();
 
+#if DML_TARGET_VERSION >= 0x6300
+        DML_RESAMPLE3_OPERATOR_DESC operatorDesc = {};
+        operatorDesc.Antialiased = static_cast<BOOL>(antialiased);
```

Comment:
>Antialiased [](http://example.com/codeflow?start=21&length=11)

Huh, thought we went with IsAntialiased. The majority of our API BOOL's use a verb form. e.g.

```
    BOOL UseClipThreshold;
    BOOL HasExclusiveSum;
    BOOL NormalizeVariance;
    BOOL IncludePadding;
    BOOL HasExclusiveProduct;
    BOOL AlignRegionsToCorners;
    BOOL Antialiased;
    BOOL UseMean;
    BOOL UseVariance;
```

Shoot. Oh well, too late now. (resolve me)